src/components/notifications/SendGrid.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+    <div class="mb-3">
+        <label for="sendgrid-api-key" class="form-label">{{ $t("SendGrid API Key") }}</label>
+        <HiddenInput id="push-api-key" v-model="$parent.notification.sendgridApiKey" :required="true" autocomplete="new-password"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <label for="sendgrid-from-email" class="form-label">{{ $t("From Email") }}</label>
+        <input id="sendgrid-from-email" v-model="$parent.notification.sendgridFromEmail" type="email" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="sendgrid-to-email" class="form-label">{{ $t("To Email") }}</label>
+        <input id="sendgrid-to-email" v-model="$parent.notification.sendgridToEmail" type="email" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="sendgrid-cc-email" class="form-label">{{ $t("smtpCC") }}</label>
+        <input id="sendgrid-cc-email" v-model="$parent.notification.sendgridCcEmail" type="email" class="form-control">
+        <div class="form-text">{{ $t("Separate multiple email addresses with commas") }}</div>
+    </div>
+    <div class="mb-3">
+        <label for="sendgrid-bcc-email" class="form-label">{{ $t("smtpBCC") }}</label>
+        <input id="sendgrid-bcc-email" v-model="$parent.notification.sendgridBccEmail" type="email" class="form-control">
+        <small class="form-text text-muted">{{ $t("Separate multiple email addresses with commas") }}</small>
```

Comment:
Please add this key as well to `en.json`

![image](https://github.com/user-attachments/assets/e3cec35c-b226-473e-bb44-858d4af70732)

Reply:
done 
0433802